use-timescale/migration/dual-write-and-backfill.md

```diff
@@ -429,9 +452,28 @@ timescaledb-parallel-copy \
 
 ### 7. Enable background jobs
 
-TODO: what about continuous aggregates?
+Before enabling the jobs, verify if any continuous aggregate refresh policies
```

Comment:
@jgpruitt These are the instructions for the refresh cagg scripts. WDYT?

It's still missing how to handle the jobs in source. We can try initially with asking customers to disable them in source, refresh and then enable both source and target.

If the process takes too long, maybe an option is to run the scripts twice. First with the jobs on source enable, then again with the jobs on source disable. The second time should be faster, we expect the backfill to take much longer than the refresh, running it a second time the refresh window I'd expect to be smaller. Maybe most of them won't need to be updated on the second run, reducing the time the jobs would have to be disable in source (if that's a concern).

I'm thinking that instructions or support to refresh caggs one by one might also be a good idea, just to have feature parity with the backfill tool `--filter`.

Reply:
I *think we should start out by advising that background jobs be left running on the source and disabled on the target during backfill. If we see folks run into issues we can adjust.

I also agree that running the refresh script twice might solve the issue if we run into it.

Adding a `--filter` option to the script shouldn't be difficult to do. Sounds good to me